src/docs/design/design-resources.mdx

```diff
@@ -0,0 +1,28 @@
+import { LinkGrid } from '../../app/components';
+import { patternLinks, figmaCommunityFileLinks, figmaPluginLinks } from '../../__configuration__/design';
+
+<!-- KEYWORDS: design resources -->
+
+# Design Resources
+
+## Guidelines
+
+PX Blue has a few design guidelines that serve as the foundation of the design system. They are intentional design choices we made to shape the PX Blue brand — what PX Blue applications should look like, and how PX Blue applications would interact with their users.
+
+<LinkGrid links={patternLinks} style={{ marginTop: 16 }} />
```

Comment:
I would leave this section out (see comment below about moving the reference to the style guide into the intro page) since it's not exactly a _resource_ and we already have a top level page dedicated to the style stuff and patterns.